inspirehep/modules/theme/jinja2filters.py

```diff
@@ -598,8 +598,7 @@ def publication_info(record):
             if 'conference_record' in pub_info:
                 conference_rec = replace_refs(pub_info['conference_record'],
                                               'es')
-                if conference_rec and conference_rec.get('control_number'):
-                    conference_recid = conference_rec['control_number']
+                conference_recid = conference_rec['control_number']
```

Comment:
watch out, seems like the line got deleted by merging. This is causing QA search page to break.